04/index.html

```diff
@@ -5,26 +5,33 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" href="global.css">
+    <link rel="stylesheet" media="(max-width: 600px)" href="mobile.css">
+    <link rel="stylesheet" media="(min-width: 601px)" href="normal.css">
 </head>
 <body>
-    
+  <header>
+    <label for="menu-switcher"></label>
```

Comment:
W wersji mobilnej nie działa rozwijanie menu zagnieżdzonego.
Aby sprawdzić działanie musisz przejść do emulatora wersji mobilnej:
![image](https://user-images.githubusercontent.com/55095257/180384446-bfaf8a7f-2ddb-49e0-a179-b69335306b58.png)

Najprościej po prostu pokazać wszystkie opcje od razu.

Reply:
U mnie działa :) Mam na myśli to, że po kliknięciu menu się rozwija prawidłowo. Tylko kod jest bardzo chaotycznie napisany dlatego layout się poprzestawiał, nie umiem tego naprawić. W aktualnym tasku mam podobną rzecz do wykonania, tam spróbuję wykonać ją prawidłowo.